uPortal-events/src/main/java/org/apereo/portal/events/rest/AnalyticsPortalEventsService.java

```diff
@@ -0,0 +1,28 @@
+package org.apereo.portal.events.rest;
+
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.apereo.portal.events.IPortalAnalyticsEventFactory;
+import org.apereo.portal.security.IPerson;
+import org.apereo.portal.security.IPersonManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AnalyticsPortalEventsService {
```

Comment:
Could you add an interface here so adopters can overlay this logic with a customized flow in uPortal-start?